File: Taller2EDchiang/Fecha.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   Fecha.cpp
 * Author: Usuario
 * 
 * Created on 17 de noviembre de 2017, 0:13
 */

#include "Fecha.h"

Fecha::Fecha() {
}

Fecha::Fecha(int hora,int minutos) {
    this->hora=hora;
    this->minutos=minutos;

}

int Fecha::getHora() {
    return this->hora;
}

int Fecha::getMinutos() {
    return this->minutos;
}

void Fecha::setHora(int hora) {
    this->hora=hora;
}

void Fecha::setMin(int minutos) {
    this->minutos=minutos;
}







Fecha::~Fecha() {
}

File: Taller2SGOSmestre/src/taller2sgosmestre/Fecha.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package taller2sgosmestre;

/**
 *
 * @author Usuario
 */
public class Fecha {
    private int dia;
    private int mes;
    private int año;
    private Prestamo prestamo;

    public Fecha(int dia, int mes, int año) {
        this.dia = dia;
        this.mes = mes;
        this.año = año;
       
    }

    public Prestamo getPrestamo() {
        return prestamo;
    }

    public void setPrestamo(Prestamo prestamo) {
        this.prestamo = prestamo;
    }

    public int getDia() {
        return dia;
    }

    public void setDia(int dia) {
        this.dia = dia;
    }

    public int getMes() {
        return mes;
    }

    public void setMes(int mes) {
        this.mes = mes;
    }

    public int getAño() {
        return año;
    }

    public void setAño(int año) {
        this.año = año;
    }

 
    
}
